.config/cargo_spellcheck.dic

```diff
@@ -57,8 +58,10 @@ stdout
 tuple
 unordered
 untyped
+v3
```

Comment:
[This is how it's done in Polkadot as well.](https://github.com/paritytech/polkadot/blob/master/scripts/gitlab/lingua.dic#L241-L243)